docs/BestPractices.md

```diff
@@ -39,6 +39,29 @@ FROM node:7.3.0-alpine
 RUN addgroup -S node && adduser -S -g node node 
 ```
 
+Note that the `node` user is neither a build-time nor a run-time dependency and
```

Comment:
I am a big fan of max length myself, but the rest of *this* document does not have any max length.